Minio/V4Authenticator.cs

```diff
@@ -114,7 +114,11 @@ public void Authenticate(IRestClient client, IRestRequest request)
             DateTime signingDate = DateTime.UtcNow;
             this.SetContentMd5(request);
             this.SetContentSha256(request);
-            this.SetHostHeader(request, client.BaseUrl.Host + ":" + client.BaseUrl.Port);
+            if (client.BaseUrl.Port == 80 || client.BaseUrl.Port == 443) {
+                this.SetHostHeader(request, client.BaseUrl.Host);
```

Comment:
@kannappanr Can we add a comment saying that default ports need not be included for the signature calculation?